### PR TITLE
WW-360 initial container restyling

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 1]
```diff
@@ -9,11 +9,20 @@ $color-borders: #CECECE;
...
+.skin-oasis.page-Special_CreateNewWiki {
+	background-color: $wds-color-off-white;
+}
+
 .WikiaPage {
 	border: 0;
 
 	.WikiaPageBackground {
-		@include linear-gradient(false, top, #01386e, 0%, #2185da, 25%, #fff, 60%);
+		background-color: $wds-color-off-white;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Please set background only in one place. If the background is inherited you can overwrite it by `background: none`;

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 2]
```diff
@@ -9,11 +9,20 @@ $color-borders: #CECECE;
 
 @import 'extensions/wikia/ThemeDesigner/css/ThemeTab';
 
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/_colors.scss';
+
+$normal-spacing: 24px;
```
Comment (sebastianmarzjan) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Can we keep the same convention as in community page, i.e.
$content-spacing-.\* ?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 3]
```diff
@@ -31,14 +40,14 @@ $color-borders: #CECECE;
 
 #CreateNewWiki {
 	background: white;
-	border: 10px solid #8ac342;
+	border: 1px solid $wds-color-faint-gray;
 	color: #747474;
```
Comment (sebastianmarzjan) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss:37]: Is this color in DS? If not, maybe it needs to be adjusted for compliance?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 4]
```diff
@@ -9,11 +9,20 @@ $color-borders: #CECECE;
 
 @import 'extensions/wikia/ThemeDesigner/css/ThemeTab';
 
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/_colors.scss';
+
+$normal-spacing: 24px;
+$cnw-size: 700px;
+
+.skin-oasis.page-Special_CreateNewWiki {
```
Comment (lukaszkonieczny) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: These classes are auto-generated. Maybe we can use for example `.WikiaSiteWrapper` here?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 5]
```diff
@@ -31,14 +40,14 @@ $color-borders: #CECECE;
...
 	background: white;
-	border: 10px solid #8ac342;
+	border: 1px solid $wds-color-faint-gray;
 	color: #747474;
 	font-size: 12px;
 	font-weight: normal;
-	margin: 50px auto 0;
-	padding: 30px 30px 60px 75px;
+	margin: 4px auto 0;
+	padding: 21px 0 60px;
```
Comment (sebastianmarzjan) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Are those paddings and margin from Design System? Especially 21px looks strangely specific

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 6]
```diff
@@ -31,14 +31,15 @@ $color-borders: #CECECE;
 
 #CreateNewWiki {
 	background: white;
-	border: 10px solid #8ac342;
+	box-sizing: border-box;
+	border: 1px solid $wds-color-faint-gray;
 	color: #747474;
```
Comment (sebastianmarzjan) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss:37]: What about this color? Is it a DS one?

**Comment by sebastianmarzjan** [comment]
Overall seems fine, though it's unclear to me why only some and not all 

```
@if ($responsive-or-breakpoints-layout-enabled) {

}
```

blocks were removed

**Comment by lukaszkonieczny** [comment]
And one my comment disappears ;)

https://github.com/Wikia/app/pull/11763#discussion_r86330183

**Comment by lukaszkonieczny** [comment]
I wonder can we just remove `$responsive-or-breakpoints-layout-enabled` at all? It's based on some global variables like `wgOasisBreakpoints`, `wgOasisGrid` or `wgOasisResponsive` and we may be able to check them while Create New Wiki process is run only on one specific wiki which is `wikia.com`. Also I suppose that `wgOasisResponsive` is not longer supported after me moved to breakpoints but there is a lot of residues in the code

**Review by lukaszkonieczny — APPROVED** [review]

**Pull request merged**